src/content/structured/community/roadmap.mdx

```diff
@@ -24,37 +24,34 @@ We welcome contributions of code, issues raised and more. Read about [how to con
 
 ## New functionality added
 
-- [Toast](/components/toast) component.
-- Updates to the [card](/components/card) component, with images and expandable area.
-- [Compact stepper](/components/stepper) component.
-- [Chips](/components/chip) component.
-- [Popover menu](/components/popover-menu) component.
-- Added asynchronous filtering for [search bar](/components/search-bar) component.
+- Open sourcing our Figma UI Kit to Figma Community.
+- Research and implement best practice for sustainability improvements.
+- Releasing accordian.
+- Releasing dialog.
```

Comment:
```suggestion
- [Dialog](https://design.sis.gov.uk/components/dialog) component.
```
In addition to the @GCHQ-Developer-530's suggested change, I don't think we need the word "releasing" before the component as these components have already been released. In the past, we have just stated the component